src/overmap.cpp

```diff
@@ -713,52 +712,52 @@ bool overmap::has_note(int const x, int const y, int const z) const
         return false;
     }
 
-    for (auto &i : layer[z + OVERMAP_DEPTH].notes) {
```

Comment:
Fair enough. I tend to think that being explicit with a named algorithm is better though.
A wonderful [talk](http://channel9.msdn.com/Events/GoingNative/2013/Cpp-Seasoning) given about a year ago on the matter. A good watch regardless of this particular case.